portal/v2/src/ClusterDetailListComponents/ClusterOperator.tsx

```diff
@@ -0,0 +1,85 @@
+import { Component } from "react"
+import { Stack, Text, PivotItem, IStackItemStyles } from '@fluentui/react';
+import { ICondition, IClusterOperator } from "./ClusterOperatorsWrapper";
+import { contentStackStylesNormal } from "../App";
+import { MultiInfoList } from "./InfoList"
+
+export interface ClusterOperatorComponentProps {
+    clusterOperators: any
```

Comment:
Are we able to define a concrete type here? If not `unknown` would be a better option.